libs/community/langchain_community/embeddings/voyageai.py

```diff
@@ -121,7 +121,7 @@ def _get_embeddings(
         embeddings: List[List[float]] = []
 
         if batch_size is None:
-            batch_size = self.batch_size
```

Comment:
we probably don't want to change this but instead put this logic in validate_environment to change self.batch_size?

```python
    @root_validator(pre=True)
    def validate_environment(cls, values: Dict) -> Dict:
        """Validate that api key and python package exists in environment."""
        values["voyage_api_key"] = convert_to_secret_str(
            get_from_dict_or_env(values, "voyage_api_key", "VOYAGE_API_KEY")
        )
        if "batch_size" not in values:
            values["batch_size"] = 72 if values["model"] in ...
        return values
```

Reply:
L96

Reply:
No, I meant when is it called? There is no place calling this function in this file. So I am assuming it is called somewhere in its base class?

Reply:
nvm, it is a Pydantic thing. Yes, it looks like a good idea to do it here. Thanks!